### PR TITLE
Add disclaimer about language server complexity

### PR DESCRIPTION
**Pull request by christoph-fricke**
Adds a small disclaimer to the root README and extension README to inform the user about the complexity of building a KLighD powered language server.

This has been an advice by @a-sr since it is quite complex to build a language server. Users that want to integrate the extension might not be aware of that.

### CODE REVIEW

**Comment by christoph-fricke** [comment]
@a-sr Does this change cover the idea that you had in mind?

File: README.md [hunk 1]
```diff
@@ -26,14 +26,22 @@ Development of this project requires [Node.js _v14.x_](https://nodejs.org) and
...
-placed as a jar named `language-server.jar` in the _applications/klighd-cli_ folder.
+Furthermore, development of the `klighd-cli` requires a language server with
+[KLighD](https://github.com/kieler/KLighD) synthesis capabilities. The server can either be started
+separately, using a socket for communication, or placed as a jar named `language-server.jar` in the
+_applications/klighd-cli_ folder.
+
+#### Disclaimer
+
+Developing a language server that uses [KLighD](https://github.com/kieler/KLighD) to fulfill all
+requirements to be usable with these clients is no easy tasks. Until the distribution of
```
Comment (christoph-fricke) [inline README.md:42]:
```suggestion
requirements to be usable with these applications is no easy task. Until the distribution of
```

File: applications/klighd-vscode/README.md [hunk 2]
```diff
@@ -21,6 +21,13 @@ Instead, it should be used as a dependency by other extensions to easily support
 visualization with KLighD. Your host extension is responsible for configuring a language client,
 while the KLighD extension handles everything related to diagrams.
 
+### Disclaimer
+
+Developing a language server for your extension that uses [KLighD](https://github.com/kieler/KLighD)
+to fulfill all requirements to be usable with this extension is no easy tasks. Until the distribution of
```
Comment (christoph-fricke) [inline applications/klighd-vscode/README.md:27]:
```suggestion
to fulfill all requirements to be usable with this extension is no easy task. Until the distribution of
```

File: README.md [hunk 3]
```diff
@@ -26,14 +26,22 @@ Development of this project requires [Node.js _v14.x_](https://nodejs.org) and
...
+[KLighD](https://github.com/kieler/KLighD) and documentation about building your own language server
+is improved, feel free to seek advise from a member of the KIELER working group.
```
Comment (christoph-fricke) [inline README.md:44]:
```suggestion
is improved, feel free to seek advice from a member of the KIELER working group.
```

File: applications/klighd-vscode/README.md [hunk 4]
```diff
@@ -21,6 +21,13 @@ Instead, it should be used as a dependency by other extensions to easily support
...
+[KLighD](https://github.com/kieler/KLighD) and documentation about building your own language server
+is improved, feel free to seek advise from a member of the KIELER working group.
```
Comment (christoph-fricke) [inline applications/klighd-vscode/README.md:29]:
```suggestion
is improved, feel free to seek advice from a member of the KIELER working group.
```

**Review by christoph-fricke — COMMENTED** [review]
Suggestions from @NiklasRentzCAU

**Review by NiklasRentzCAU — APPROVED** [review]
LGTM.

**Pull request merged**